splunk-otel-android/src/main/java/com/splunk/rum/SplunkScreenNameExtractor.java

```diff
@@ -19,6 +19,7 @@
 import android.app.Activity;
 import androidx.annotation.Nullable;
 import androidx.fragment.app.Fragment;
+import io.opentelemetry.rum.internal.instrumentation.RumScreenName;
```

Comment:
Well, it's a bummer that a test didn't catch this -- but I believe that this breaks existing functionality for users using the splunk annotation. I think we need to take this back out in the short term, so that the fallback mechanism can still operate.